File: Advanced Methods of Programming/JavaClientServer - Spring/Model/src/main/java/Model/Participant.java
package Model;

import java.io.Serializable;

public class Participant implements HasId<Integer>, Serializable {
    private Integer idParticipant;
    private String numeParticipant;
    private Integer varstaParticipant;

    public Participant(Integer idParticipant, String numeParticipant, Integer varstaParticipant){
        this.idParticipant=idParticipant;
        this.numeParticipant=numeParticipant;
        this.varstaParticipant=varstaParticipant;
    }

    @Override
    public Integer getId() { return this.idParticipant; }

    @Override
    public void setId(Integer integer) { this.idParticipant=integer; }

    public String getNumeParticipant() { return numeParticipant; }

    public void setNumeParticipant(String numeParticipant) { this.numeParticipant = numeParticipant; }

    public Integer getVarstaParticipant() { return varstaParticipant; }

    public void setVarstaParticipant(Integer varstaParticipant) { this.varstaParticipant = varstaParticipant; }

    public String toString(){ return idParticipant+" "+numeParticipant+" "+varstaParticipant; }
}

File: Advanced Methods of Programming/Client-ServerCSharp/SharedDLL/Domain/Participant.cs
using System;

namespace SharedDLL.Domain
{
    [Serializable]
    public class Participant : IHasID<int>
    {
        private int IdParticipant;
        private string NumeParticipant;
        private int VarstaParticipant;

        public Participant(int Id,string Nume,int Varsta)
        {
            this.IdParticipant = Id;
            this.NumeParticipant = Nume;
            this.VarstaParticipant = Varsta;
        }
        
        public int GetId() { return this.IdParticipant; }

        public string GetNumeParticipant() { return this.NumeParticipant; }

        public int GetVarstaParticipant() { return this.VarstaParticipant; }
    }
}